Makefile

```diff
@@ -291,14 +299,12 @@ $(USERGUIDE_ZIP_FILE): $(USERGUIDE_TARGET_DIR)
 # know which data zipfiles to create until the data repo is cloned.
 # All data zipfiles are written to dist/data/*.zip
 ZIPDIRS = Annual_Water_Yield \
```

Comment:
Does this list need to include stormwater also?

Reply:
Well shit. We didn't include the Stormwater model's sample data in the release.

Reply:
The model is also missing in `invest-autotest.py`

Reply:
I made an issue for this and related things. #808 . I don't think we necessarily need to fix it in this PR. It might be easier to merge this one and then do another so we can clearly see the changes related to #808.